episode-6.msrt

```diff
@@ -387,7 +391,8 @@
 78
 00:05:06,020 --> 00:05:10,110
 [ru] [саркастически] Эх, радуйся, Кэррот. Похоже, никто не увидит твое чУдное зелье..
-[en] [sarcastically] You're lucky, Carrot; It seems nobody will see your wondrous potion...
+[en] [Version 1:] [sarcastically] Ah, be glad Carrot. It seems nobody will see your wondrous potion...
```

Comment:
@scribblemaniac I like the idea with "It's your lucky day, Carrot". But in my opinion we still have to focus on Carrot, since Pepper assumes it was *his* mistake to take a wrong portion.
Maybe something like this: "It's your lucky day, Carrot - seems like nobody will see your wondrous potion..."
Again, grammatical corrections are appreciated. ^__^